app/code/Magento/Eav/Model/Entity/Attribute/Source/AbstractSource.php

```diff
@@ -166,4 +166,22 @@ public function toOptionArray()
     {
         return $this->getAllOptions();
     }
+
+    /**
+     * Multibyte support strcasecmp function version
+     * @param string $str1
+     * @param string $str2
+     * @param null|string $encoding
+     * @return int|\\lt
+     */
+    private function mbStrcasecmp($str1, $str2, $encoding = null)
```

Comment:
Hello,
why do you add an optional parameter if it's never used? Or am I missing something?
Thanks!

Reply:
Hello @gomenca, do you have any update on this? Thanks!

Reply:
Hi @aleron75 , you are right. I've just used this function in another places and used this parameter. I drop it.